src/cascadia/TerminalSettingsEditor/Appearances.cpp

```diff
@@ -573,11 +573,9 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         }
     }
 
-    double AppearanceViewModel::LineHeight() const
+    double AppearanceViewModel::_extractCellSizeValue(const hstring val) const
```

Comment:
```suggestion
    double AppearanceViewModel::_extractCellSizeValue(const hstring& val) const
```

technically best practice